mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -138,9 +157,11 @@ def _extract_run_id(artifact_uri):
         artifact_path = extract_and_normalize_path(artifact_uri)
         return artifact_path.split("/")[3]
 
-    def _call_endpoint(self, service, api, json_body):
+    def _call_endpoint(self, service, api, json_body, path_params=None):
         db_creds = get_databricks_host_creds(self.databricks_profile_uri)
         endpoint, method = _SERVICE_AND_METHOD_TO_INFO[service][api]
+        if path_params:
+            endpoint = endpoint.format(**path_params)
```

Comment:
The endpoints for trace upload and download look like `"/mlflow/traces/{trace_id}/credentials-for-data-{download|upload}"`. `trace_id` needs to be replaced.